docs/docs/guides/tracking/tracing.mdx

```diff
@@ -748,4 +739,64 @@ Calculated Fields:
     * Duration
     * Status
 
+## FAQs
+
+### How do I disable tracing?
+
+**Environment Variable:**
+
+In situations where you want to unconditionally disable tracing for the entire program, you can set the environment variable `WEAVE_TRACING_ENABLED` to `false`.
+
+**Client Initialization:**
+
+Sometimes, you may want to conditionally enable tracing for a specific initialization based on some condition. In this case, you can initialize the client with the `tracing_enabled` flag.
```

Comment:
I don't think this is correct. Isn't it `weave.init(... settings={"disabled": True})`

Reply:
🤦 thank you